core/src/idx/trees/store/mod.rs

```diff
@@ -29,7 +29,7 @@ where
 	N: TreeNode + Debug + Clone,
 {
 	/// caches every read nodes, and keeps track of updated and created nodes
-	Write(TreeWrite<N>),
+	Write(Box<TreeWrite<N>>),
```

Comment:
I prefer to exempt this line until we check that this change has no impact on performances

Reply:
Makes sense, thanks Emmanuel! I have reverted the change and added an exemption in 493652b.